homeassistant/components/lacrosse_view/sensor.py

```diff
@@ -44,9 +45,14 @@ class LaCrosseSensorEntityDescription(
     """Description for LaCrosse View sensor."""
 
 
-def get_value(sensor: Sensor, field: str) -> float:
+def get_value(sensor: Sensor, field: str) -> float | int | str:
     """Get the value of a sensor field."""
-    return float(sensor.data[field]["values"][-1]["s"])
+    value = sensor.data[field]["values"][-1]["s"]
+    try:
+        value = float(value)
+    except ValueError:
+        return str(value)  # str
+    return int(value) if value.is_integer() else value  # int, else float
```

Comment:
```suggestion
    return int(value) if value.is_integer() else value
```
No need to comment something that is already obvious from the code.